bounties/npm/tagged-git-commits/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2020-06-30",
+  "AffectedVersionRange": "*",
+  "Summary": "RCE on \"tagged-git-commits\"",
+  "Author": {
+    "Username": "Mik317",
+    "Name": "Michele Romano"
+  },
+  "Package": {
+    "Registry": "npm",
+    "Name": "tagged-git-commits",
+    "URL": "https://www.npmjs.com/package/tagged-git-commits"
+  },
+  "CWEs": [
+    {
+      "ID": "78",
+      "Description": "OS command injection"
+    }
+  ],
+  "CVSS": {
+    "Version": "3.1",
+    "AV": "",
+    "AC": "",
+    "PR": "",
+    "UI": "",
+    "S": "",
+    "C": "",
+    "I": "",
+    "A": "",
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/aichbauer/node-tagged-git-commits",
+    "Codebase": [
+      "NodeJS"
+    ]
+  },
+  "Permalinks": [
+    "https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L29",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L30",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L32",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L33",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L39",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L40",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L42",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L43"
```

Comment:
Remove space at the beginning of the string

Reply:
Done